sycl/test-e2e/syclcompat/launch/launch_policy.cpp

```diff
@@ -0,0 +1,328 @@
+/***************************************************************************
+ *
+ *  Copyright (C) Codeplay Software Ltd.
+ *
+ *  Part of the LLVM Project, under the Apache License v2.0 with LLVM
+ *  Exceptions. See https://llvm.org/LICENSE.txt for license information.
+ *  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCLcompat API
+ *
+ *  launch_config.cpp
+ *
+ *  Description:
+ *     launch<F> with config tests
+ **************************************************************************/
+
+// RUN: %clangxx -std=c++20 -fsycl -fsycl-device-code-split=per_kernel -fsycl-targets=%{sycl_triple} %s -o %t.out
+// RUN: %{run} %t.out
+
+#include "sycl/ext/intel/experimental/kernel_execution_properties.hpp"
```

Comment:
Over all the tests the usage of `"` and `<>` in the includes is rather inconsistent.
I think overall we should look for global include paths for any `sycl` and `syclcompat` headers, and the local path for any helpers used in the tests (fixtures and such)

Reply:
Thanks good point :+1: fixed